packages/apps/app-dashboard/src/components/consent/hooks/useParameterManagement.ts

```diff
@@ -180,44 +176,6 @@ export const useParameterManagement = ({
     fetchExistingParametersRef.current = fetchExistingParameters;
   }, [fetchExistingParameters]);
 
-  useEffect(() => {
```

Comment:
This hook was re executing multiple times bringing the old app version data. Just removing it fixed the frequent old data display bug